Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -60,8 +60,6 @@ private void Update()
             gazeDirection = Camera.main.transform.forward;
 
             UpdateRaycast();
-
-            UpdateStabilizationPlane();
```

Comment:
Or, you can check if SetStabilizationPlane bool is checked and programmatically add your script from GazeManager.